Demo/AuthenticatorDemo/ViewController.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+import WordPressAuthenticator
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        print("Error domain is \(WordPressAuthenticator.errorDomain)")
```

Comment:
Just a simple `print` of a value coming from the framework to make sure everything is wired up alright.